packages/website/pages/docs/usage/configuration.mdx

```diff
@@ -71,6 +71,21 @@ t('ordered', {orderDate: parseISO('2020-11-20T10:36:01.516Z')});
 t('latitude', {latitude: 47.414329182});
 ```
 
+## Global Rich Text Elements
+To achieve consistent usage and styling of rich text elements, you can define a set of global elements and pass them to the provider.
+If you define an element locally it will override the global default.
+
+```js
+<NextIntlProvider
+  ...
+  defaultRichTextElements={{
+    b: (value) => <strong>{value}</strong>
+  }}
+>
+  <App />
+</NextIntlProvider>
+```
+
```

Comment:
> BTW is one of use-intl's tests broken due to a spelling mistake (plural and singular), I can also create a PR to fix it if you like.

Which is the test you're seeing problems with? Seems like CI passes. I'd be curious if it's a problem related to the environment. You might need node>=14, this is currently not enforced.

Reply:
Interestingly enough now the CI pipeline failed... locally it failed the other way around. Can you maybe somehow undo/exclude the [34d9c05](https://github.com/amannn/next-intl/pull/86/commits/34d9c05817bbfdcccf66d3add66fc0828b2d715e) commit.

Reply:
I think if you run `git revert 34d9c05` in your branch that should do the trick.